run-custom-mao.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash -xe
+
+# This script needs an machine-api-operator image as an input parameter
+# For example: ./run-custom-mao.sh quay.io/sdasu/machine-api-operator:sdasu-fix
+export SCRIPTS_DIR=$(pwd)
```

Comment:
This won't be a useful value, since the value will change depending on where the calling shell is running - you can use SCRIPTDIR as defined in common.sh (or copy the same approach).

This isn't currently used, but probably should be instead of the relative paths below?